datajunction-ui/src/app/pages/NodePage/index.jsx

```diff
@@ -26,17 +27,17 @@ export function NodePage() {
   const { name, tab } = useParams();
 
   const [state, setState] = useState({
-    selectedTab: tab || 'info',
+    selectedTab: tab || "info",
```

Comment:
Hmm, do we have different lint settings? I typically run the following linters:
```
yarn prettier . --write
yarn lint --fix
```

Reply:
This is probably a silly thing. I'll run your lints